framework/src/application/node/node.ts

```diff
@@ -474,6 +480,16 @@ export class Node {
 					params.data,
 					params.peerId,
 				),
+			getSchema: () => ({
+				account: this._chain.accountSchema,
+				baseBlockHeader: blockSchema,
+				blockHeaders: {
+					0: BlockProcessorV0.schema,
+					2: BlockProcessorV2.schema,
+				},
+				baseTransaction: BaseTransaction.BASE_SCHEMA,
```

Comment:
```suggestion
				transaction: BaseTransaction.BASE_SCHEMA,
```

Reply:
I think base transaction is ok here why change as it's literally base transaction schema?